suites/message/nested.go

```diff
@@ -359,6 +359,65 @@ func MessageTest_NestedSends(t *testing.T, factory state.Factories) {
 
 	})
 
+	t.Run("fail undecodable params nested send", func(t *testing.T) {
+		puppetBalance := big.NewInt(1_000)
+		td := builder.WithActorState(drivers.ActorState{
+			Addr:    PuppetAddress,
+			Balance: puppetBalance,
+			Code:    puppet.PuppetActorCodeID,
+			State:   &puppet.State{},
+		}).Build(t)
+		defer td.Complete()
+
+		// user separate actors for easier balance assertions
+		alice, _ := td.NewAccountActor(drivers.SECP, acctDefaultBalance)
+		bob, _ := td.NewAccountActor(drivers.SECP, acctDefaultBalance)
+
+		result1 := td.ApplyMessage(
+			td.MessageProducer.PuppetSendMarshalCBORFailure(alice, PuppetAddress, &puppet.SendParams{
```

Comment:
cause the puppet actor to invoke an inner send with undecodable params: https://github.com/filecoin-project/specs-actors/blob/master/actors/puppet/puppet.go#L75